x/group/keeper/msg_server.go

```diff
@@ -345,12 +344,8 @@ func (k Keeper) CreateGroupPolicy(goCtx context.Context, req *group.MsgCreateGro
 			// next sequence value and derive a new address.
 			continue
 		}
-		acc := k.accKeeper.NewAccount(ctx, &authtypes.ModuleAccount{
```

Comment:
btw, a good addition to this PR would be to have `NewAccount` call `.Validate()`, this way only valid accounts go in state